.travis.yml

```diff
@@ -2,15 +2,19 @@ language: node_js
 node_js:
 - 12
 script:
-- yarn travis
+- npm run build
```

Comment:
Why did you switch to `npm`? If there's an important reason, then we need to delete `yarn.lock` and add a `package-lock.json`.

Reply:
That was unintentional (I was copying config from another project)